src/Raven.Studio/typescript/components/pages/database/settings/revisionsBinCleaner/RevisionsBinCleaner.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { databaseAccessArgType, withBootstrap5, withStorybookContexts } from "test/storybookTestUtils";
+import { Meta, StoryObj } from "@storybook/react";
+import RevisionsBinCleaner from "components/pages/database/settings/revisionsBinCleaner/RevisionsBinCleaner";
+import { mockServices } from "test/mocks/services/MockServices";
+import { mockStore } from "test/mocks/store/MockStore";
+import React from "react";
+import { DatabasesStubs } from "test/stubs/DatabasesStubs";
+
+export default {
+    title: "Pages/Database/Settings/Revisions Bin Cleaner",
+    component: RevisionsBinCleaner,
+    decorators: [withStorybookContexts, withBootstrap5],
+} satisfies Meta<typeof RevisionsBinCleaner>;
```

Comment:
`component` will add all props to control

```suggestion
export default {
    title: "Pages/Database/Settings/Revisions Bin Cleaner",
    decorators: [withStorybookContexts, withBootstrap5],
} satisfies Meta;
```

Reply:
Done